video/quickstart.rb

```diff
@@ -13,39 +13,38 @@
 # limitations under the License.
 
 # [START videointelligence_quickstart]
-require "google/cloud/video_intelligence/v1beta1"
+require "google/cloud/video_intelligence"
 
-video_client = Google::Cloud::VideoIntelligence::V1beta1::VideoIntelligenceServiceClient.new
+video_client = Google::Cloud::VideoIntelligence.new
 features     = [:LABEL_DETECTION]
 path         = "gs://demomaker/cat.mp4"
 
 # Register a callback during the method call
-operation = video_client.annotate_video path, features do |operation|
+operation = video_client.annotate_video input_uri: path, features: features do |operation|
```

Comment:
I saw you inlined the value for features for other samples, but not this one. Is it because the line is long?

Reply:
Would you like this changed? Please let me know either way as soon as you have a moment